mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_2483 net/minecraft/nbt/AbstractNbtList
 	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
-	METHOD add (ILjava/lang/Object;)V
+	METHOD add add (ILjava/lang/Object;)V
```

Comment:
I must admit that I do not understand how this actually occurs, I only made the change using Enigma in both NbtByteArray and NbtLongArray, and it would turn out this is the change that resulted from it